lib/model/src/main/java/io/atlasmap/v2/Append.java

```diff
@@ -1,34 +1,50 @@
 package io.atlasmap.v2;
```

Comment:
This is an example of what the Append action looked like before this PR. It contains only its parameter definition and accessor/mutator.